.gitignore

```diff
@@ -6,3 +6,4 @@ cmake-build-debug/
 .idea/
 
 /compile
+
```

Comment:
Now you have introduced an extra newline...

Reply:
done